electron/src/renderer/preload-webview.ts

```diff
@@ -103,22 +103,16 @@ const subscribeToWebappEvents = () => {
     ipcRenderer.sendToHost(EVENT_TYPE.ACCOUNT.UPDATE_INFO, info);
   });
 
-  window.addEventListener(
-    'NavigationEvent',
-    event => {
-      const accountId = new URLSearchParams(window.location.search).get('id');
-      const data = (event as CustomEvent).detail;
-      if (data) {
-        ipcRenderer.send(EVENT_TYPE.WRAPPER.CUSTOM_WEBAPP, {
-          accountId,
-          customURL: data.url,
-        });
-      }
-    },
-    {
-      once: true,
-    },
-  );
+  window.addEventListener('NavigationEvent', event => {
+    const accountId = new URLSearchParams(window.location.search).get('id');
+    const data = (event as CustomEvent).detail;
+    if (data) {
+      ipcRenderer.send(EVENT_TYPE.WRAPPER.CUSTOM_WEBAPP, {
```

Comment:
Would be nice if the naming of the internal event would match the name of the external event (NavigationEvent)